ui/src/components/ApiError/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { QueryErrorType, getMessageFromApiError } from "~/api/errorHandling";
+
+import Alert from "~/design/Alert";
+import { useTranslation } from "react-i18next";
+
+const ApiError = ({
+  error,
+  label,
+}: {
+  error: QueryErrorType;
+  label?: string;
+}) => {
+  const { t } = useTranslation();
+  const errorMessage = getMessageFromApiError(error);
```

Comment:
I added different messages for the different use cases with and without an error message in 5ebdc18c7. To avoid an explosion in complexity, I decided to remove the ability to customize the error message. Now the component is a bit less customizable, but easy to use (otherwise we would need to consider messages for the different use cases in every parent component).